charts/bundler/values.yaml

```diff
@@ -42,8 +42,8 @@ ingress:
   hosts:
     - host: authz.diamond.ac.uk
       paths:
-        - path: /
-          pathType: Prefix
+        - path: /bundle.tar.gz
+          pathType: ImplementationSpecific
```

Comment:
I thought Diamond Kyverno policy disallowed this?

Reply:
There is an exception which can be granted